coffee_demo_moves:
  - gripper3/cup_open
  - arm1_3/kettle_init_cup_grab
  - gripper1/kettle_open
  - gripper3/cup_close
  - arm1_3/kettle_grab_cup_lift
  - gripper1/kettle_close
  - arm1_3/kettle_levitate_cup_place
  - gripper3/cup_open
  - arm1_3/kettle_inward_cup_back
  - arm1_3/kettle_move2dripper_raise
  - arm1_3/kettle_pour_dripper_init
  - sleep/kettle_pour
  - arm1/kettle_move2dripper
  - sleep/kettle_pour
  - arm1/kettle_pour
  - sleep/kettle_pour
  - arm1/kettle_move2dripper
  - sleep/kettle_pour
  - arm1/kettle_pour
  - sleep/kettle_pour
  - arm1/kettle_move2dripper
  - gripper3/dripper_open
  - arm1_3/kettle_inward_dripper_grab
  - sleep/dripper_pull
  - gripper3/dripper_close
  - arm1_3/kettle_levitate_dripper_lift_small
  - arm1_3/kettle_grab_dripper_pull2side
  - gripper1/kettle_open
  - arm3/dripper_lift_full
  - arm1_3/kettle_init_dripper_move
  - arm3/dripper_half_pour
  - arm3/dripper_pour
  - sleep/dripper_pour
  - arm3/dripper_move
  - arm3/dripper_exit
  - arm3/move_base
  - gripper3/dripper_open
  - arm3/move_back
arm1_3:
  kettle_init_cup_grab: [0.346, 0.010, 0.291, 1.57, 0.035, 0.029, 0.482, 0.0, 0.15, 0.0, 0.0, 0.0]
  kettle_grab_cup_lift: [0.483, 0.017, 0.290, 1.57, 0.120, 0.032, 0.33, -0.37, 0.230, 0, 0, -0.859]
  kettle_levitate_cup_place: [0.483, 0.017, 0.360, 1.57, 0.120, 0.032, 0.33, -0.37, 0.15, 0, 0, -0.881]
  kettle_inward_cup_back: [0.233, 0.017, 0.310, 1.57, 0.120, 0.032, 0.28, -0.314, 0.15, 0, 0, -0.881]
  kettle_move2dripper_raise: [0.000, -0.396, 0.568, 1.57, -0.041, -1.830, 0.117, -0.239, 0.172, 0, 0.762, -1.114]
  kettle_pour_dripper_init: [-0.095, -0.45, 0.555, 2.3, -0.024, -1.896, 0.044, -0.310, 0.197, 0.468, 0.036, -1.431]
  kettle_inward_dripper_grab: [0.233, 0.017, 0.310, 1.57, 0.120, 0.032, 0.058, -0.373, 0.192, 0.467, -0.036, -1.412]
  kettle_levitate_dripper_lift_small: [0.483, 0.017, 0.360, 1.57, 0.120, 0.032, 0.058, -0.363, 0.200, 0.467, -0.036, -1.412]
  kettle_grab_dripper_pull2side: [0.483, 0.017, 0.290, 1.57, 0.120, 0.032, 0.216, -0.289, 0.186, 0.287, -0.060, -0.928]
  kettle_init_dripper_move: [0.346, 0.010, 0.291, 1.57, 0.035, 0.029, 0.439, -0.18, 0.385, 0.429, -0.034, -0.461]
arm1:
  kettle_init: [0.346, 0.010, 0.291, 1.57, 0.035, 0.029]
  kettle_grab: [0.483, 0.017, 0.290, 1.57, 0.120, 0.032]
  kettle_levitate: [0.483, 0.017, 0.360, 1.57, 0.120, 0.032]
  kettle_inward: [0.233, 0.017, 0.310, 1.57, 0.120, 0.032]
  kettle_move2dripper: [0.000, -0.396, 0.568, 1.57, -0.041, -1.830]
  kettle_pour: [-0.095, -0.45, 0.555, 2.3, -0.024, -1.896]
gripper1:
  kettle_open: [0.5]
  kettle_close: [1.0]
arm3:
  cup_grab: [0.482, 0.0, 0.15, 0.0, 0.0, 0.0]
  cup_lift: [0.33, -0.37, 0.230, 0, 0, -0.859]
  cup_place: [0.33, -0.37, 0.15, 0, 0, -0.881]
  cup_back: [0.28, -0.314, 0.15, 0, 0, -0.881]
  raise: [0.117, -0.239, 0.172, 0, 0.762, -1.114]  
  dripper_init: [0.044, -0.310, 0.197, 0.468, 0.036, -1.431]
  dripper_grab: [0.058, -0.373, 0.192, 0.467, -0.036, -1.412]
  dripper_lift_small: [0.058, -0.363, 0.200, 0.467, -0.036, -1.412]
  dripper_pull2side: [0.216, -0.289, 0.186, 0.287, -0.060, -0.928]
  dripper_lift_full: [0.230, -0.339, 0.344, 0.287, -0.085, -0.974]
  dripper_move: [0.439, -0.18, 0.385, 0.429, -0.034, -0.461]
  dripper_half_pour: [0.381, -0.326, 0.355, 1.899, 0.290, -0.695]
  dripper_pour: [0.338, -0.378, 0.372, 2.318, 0.224, -0.824]
  dripper_exit: [0.500, 0.0, 0.385, 0.290, -0.077, 0.0]
  move_base: [0.500, 0.0, 0.194, 0.290, -0.077, 0.0]
  move_back: [0.450, 0.0, 0.194, 0.290, -0.077, 0.0]
gripper3:
  dripper_open: [0.85]
  dripper_close: [1.09]
  cup_open: [0.0]
  cup_close: [0.56]
sleep:
  kettle_pour: [3]
  dripper_pull: [25]
  dripper_pour: [5]